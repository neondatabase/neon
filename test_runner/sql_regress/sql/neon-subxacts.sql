DO $$
DECLARE
i numeric;
BEGIN
  create role somebody;
  FOR i IN 1..1000000 LOOP
    BEGIN
	  IF i % 1000 = 0 THEN
	    alter role somebody password 'welcome';
	  ELSE
        PERFORM 1;
	  END IF;
    EXCEPTION WHEN OTHERS THEN
      RAISE WARNING 'error';
    END;
    IF I = 1000000 THEN
      PERFORM pg_log_backend_memory_contexts(pg_backend_pid());
    END IF;
  END LOOP;
END;
$$;
