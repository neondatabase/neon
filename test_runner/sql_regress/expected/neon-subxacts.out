DO $$
DECLARE
i numeric;
BEGIN
  create role somebody;
  FOR i IN 1..1_000_000 LOOP
    BEGIN
	  IF i % 1000 = 0 THEN
	    alter role somebody password 'welcome';
	  ELSE
        PERFORM 1;
	  END IF;
    EXCEPTION WHEN OTHERS THEN
      RAISE WARNING 'error';
    END;
    IF I = 1_000_000 THEN
      PERFORM pg_log_backend_memory_contexts(pg_backend_pid());
    END IF;
  END LOOP;
END;
$$;
